app/formSchema/pages/timeMachine.ts

```diff
@@ -0,0 +1,19 @@
+const timeMachine = {
+  timeMachine: {
+    title: 'Time Mashine',
```

Comment:
```suggestion
    title: 'Time Machine',
```

Reply:
Actually I think this page just needs to be removed as I removed the rest from the schema sorry.

Reply:
Page removed